package.json

```diff
@@ -46,7 +46,8 @@
     "@rocket/cli": "^0.10.1",
     "@rocket/core": "^0.1.2",
     "@rocket/launch": "^0.6.0",
-    "@rocket/search": "^0.6.0"
+    "@rocket/search": "^0.6.0",
+    "@storybook/core": "^8.3.3"
```

Comment:
This shouldn't be needed, you should be able to depend on `storybook` instead, I hope

Reply:
`storybook` itself includes a lot of unrelated stuff. I only need `@storybook/core/channels`

Reply:
shouldn't this be in one of the packages, rather that the monorepo root?